sdcm/nemesis.py

```diff
@@ -1494,15 +1494,17 @@ def disrupt_truncate_large_partition(self):
         # do the actual truncation
         self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(ks_name, table), timeout=120)
 
-    def _modify_table_property(self, name, val, filter_out_table_with_counter=False):
+    def _modify_table_property(self, name, val, filter_out_table_with_counter=False, keyspace_table=None):
         disruption_name = "".join([p.strip().capitalize() for p in name.split("_")])
         InfoEvent('ModifyTableProperties%s %s' % (disruption_name, self.target_node)).publish()
 
-        ks_cfs = self.cluster.get_non_system_ks_cf_list(
-            db_node=self.target_node, filter_out_table_with_counter=filter_out_table_with_counter,
-            filter_out_mv=True)  # not allowed to modify MV
+        if not keyspace_table:
+            ks_cfs = self.cluster.get_non_system_ks_cf_list(
+                db_node=self.target_node, filter_out_table_with_counter=filter_out_table_with_counter,
+                filter_out_mv=True)  # not allowed to modify MV
+
+            keyspace_table = random.choice(ks_cfs) if ks_cfs else ks_cfs
```

Comment:
Let's use the nemesis_seed here for selection (or any other seed) to make sure tests are repeatable.

Reply:
done

Reply:
where?